app/views/scheduled_messages/_form.html.erb

```diff
@@ -1,7 +1,8 @@
 <%= simple_form_for @scheduled_message do |form| %>
   <%= form.input :title,
-    hint: "This is for identifying purposes only, the employee will not see the message title" %>
+    hint: "This is for identifying purposes only, the employee will not see the message title." %>
   <%= form.input :body, label: "Message body" %>
   <%= form.input :days_after_start, label: "Days after employee starts to send message" %>
+  <%= form.input :tag_list, label: "Tags", hint: "Separate tags with spaces to enter multiple tags at once." %>
```

Comment:
I did, and I was thinking ease of use in this case.  It's quicker/easier to just type a string of single words with spaces to add a bunch of tags.  There is a caveat though in that you then can't have multiple words make up a single tag.  I should ask before making that assumption:  if that is going to be a use-case then we would need to keep the comma (or some other delimiter) separation.

Furthermore, I did toy around with a small string substitution when not entering commas to put them in there (splitting the string on a space) in order to enforce multiple tags, but I wasn't a fan of it since that creates some "magic" behind the scenes (and now that it's being discussed, it makes the same assumption as noted above).  Obviously this would be okay if we explicitly states what happens, but let's nail down what behavior we're ultimately looking for.

@melodykramer: Can you please chime in with what you feel would be best?  Do you foresee adding tags that are made up of multiple words?  If so, is explicitly noting how tags should be separated (e.g., commas) in the help text enough for the app now (thinking MVP), or do you feel we would need a bit more support in the UI to prevent usage errors?  Thanks!